client/mass/test/violin.integration.spec.js

```diff
@@ -673,24 +673,30 @@ tape('test uncomputable categories legend', function(test) {
 		violin.on('postRender.test', null)
 
 		await testUncomputableCategories(violin, legendDiv)
-
+		if (test._ok) violin.Inner.app.destroy()
 		test.end()
 	}
 
 	async function testUncomputableCategories(violin, legendDiv) {
-		const categories = await detectGte({ elem: legendDiv.node(), selector: '.legend-row', count: 9 })
 		const keys = Object.keys(violin.Inner.data.uncomputableValueObj)
```

Comment:
Unless the order of legend entries is guaranteed, it is best to not hardcode the index position in which the legend entry may appear in the array as returned by `detectGte` array.